community/main.py

```diff
@@ -230,36 +230,16 @@ def on_draw(self):
 
 		# Draw the F3 Debug screen
 		if self.show_f3:
-			self.draw_f3()
+			self.f3.draw()
 
 		# CPU - GPU Sync
 		if not self.options.SMOOTH_FPS:
-			self.fences.append(gl.glFenceSync(gl.GL_SYNC_GPU_COMMANDS_COMPLETE, 0))
+			# self.fences.append(gl.glFenceSync(gl.GL_SYNC_GPU_COMMANDS_COMPLETE, 0))
```

Comment:
Can we just delete these lines if commented?

Reply:
It still serves purpose, once pyglet fixes (I hope) the missing gl function we can uncomment it.
It's a temporary solution, and if we delete it we might forget about it completely, that's why I just commented it for now.